vulture_os/applications/backend/form.py

```diff
@@ -236,24 +238,36 @@ def __init__(self, *args, **kwargs):
         """ Initialisation of fields method """
         # Do not set id of html fields, that causes issues in JS/JQuery
         kwargs['auto_id'] = False
+        mode = kwargs.pop('mode', '')
         super().__init__(*args, **kwargs)
         # Remove the blank input generated by django
         self.fields['tls_profile'].empty_label = "Plain text"
         self.fields['tls_profile'].required = False
         self.fields['source'].required = False
+        if mode == 'unix':
+            self.fields['target'].label = 'Socket'
+            del self.fields['port']
+
 
     def as_table_headers(self):
         """ Format field names as table head """
         result = "<tr><th style=\"visibility:hidden;\">{}</th>\n"
         for field in self:
+            if field.name == "mode":
+                continue
             result += "<th>{}</th>\n".format(field.label)
         result += "<th>Delete</th></tr>\n"
         return result
 
     def as_table_td(self):
         """ Format fields as a table with <td></td> """
         result = "<tr><td style=\"visibility:hidden;\">{}</td>\n".format(self.instance.id or "")
+        mode = self.instance.mode
         for field in self:
+            if field.name == 'mode':
+                continue
+            if mode == 'unix' and field.name == 'port':
+                continue
```

Comment:
Could you merge those conditions ?
if field.name == mode or (mode == unix and field.name == port):
continue